ports/psoc6/main.c

```diff
@@ -183,6 +184,7 @@ void mpy_task(void *arg) {
     mod_adc_block_deinit();
     mod_i2c_deinit();
     mod_pwm_deinit();
+    mod_spi_deinit();
     mod_pin_phy_deinit();
```

Comment:
I have the same concern here. If mod_spi_deinit ihas already deinited the physical pin, then what consequences would mod_pin_phy_deinit has if internally it is not checking a deinit operation was already done from module side.

Reply:
As before, that pin_phy_deinit() can be called several times without unexpected behavior, if deinited already, then it has no unsafe effect.